config/initializers/s3.rb

```diff
@@ -0,0 +1,13 @@
+CarrierWave.configure do |config|
+
+ # if Rails.env.production?
+  config.fog_credentials = {
+    :provider               => 'AWS', :aws_access_key_id      => ENV['S3_KEY'],
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.